amcl/src/amcl_node.cpp

```diff
@@ -919,35 +1055,49 @@ AmclNode::convertMap( const nav_msgs::OccupancyGrid& map_msg )
       map->cells[i].occ_state = +1;
     else
       map->cells[i].occ_state = 0;
+  //ROS_DEBUG("%i\n", map->cells[i].occ_state);
   }
 
   return map;
 }
 
 AmclNode::~AmclNode()
 {
-  delete dsrv_;
+  //delete dsrv_;
```

Comment:
the message filter and dynamic reconfigure server are disabled atm (but the laser_scan_sub_ should get deleted automatically from what I understand). https://github.com/ros2/navigation/pull/9/commits/e5e0c0b3ed79165116049045e05fa66b76f5b86c